### PR TITLE
feat: Implement python binding

### PR DESCRIPTION
**Pull request by jopemachine**
Related issue: https://github.com/gluesql/gluesql/issues/480.

This PR implements Python bindings for the following storages:

- [x] MemoryStorage
- [x] JsonStorage
- [x] SharedMemoryStorage
- [x] SledStorage

I think GlueSQL in Python will be a powerful tool.

Future improvements:

- [ ] Convert the query function to an asynchronous function.
- [ ] Optimize the convert_payload function for better performance.
- [ ] Add bindings for the Composite Storage.

### CODE REVIEW

File: pkg/python/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+[package]
+name = "gluesql-py"
+authors = ["Taehoon Moon <taehoon.moon@outlook.com>"]
```
Comment (panarch) [inline pkg/python/Cargo.toml:3]: you can become the first author here :)
ref.
https://github.com/gluesql/gluesql/blob/bac635e489dcf617a4f47a752ab94f6470731f71/storages/json-storage/Cargo.toml#L3-L6

Comment (jopemachine) [reply pkg/python/Cargo.toml:3]: Thanks!

**Review by panarch — COMMENTED** [review]
now it looks all great!
just before merging it, could you add a github action for python build?
we need a way to verify that other future changes not to break the python build.

**Review by panarch — APPROVED** [review]
It's so pleasure to see the new meaningful functionality.
Everything looks great!! Let's ship this to the main 👍 👍 👍

**Pull request merged**